supabase/migrations/20230815171919_base.sql

```diff
@@ -1884,9 +1884,6 @@ ALTER TABLE ONLY "public"."deleted_account"
 ALTER TABLE ONLY "public"."devices_override"
     ADD CONSTRAINT "devices_override_pkey" PRIMARY KEY ("device_id");
 
-ALTER TABLE ONLY "public"."channel_devices"
-    ADD CONSTRAINT "channel_devices_pkey" PRIMARY KEY ("device_id");
-
```

Comment:
does this change take effect? considering this already ran

Reply:
See, the thing is that I have not REALLY removed anything. This constraint is duplicated on [this line](https://github.com/Cap-go/capgo/blob/2210c90ae8c9dd957914d658b37e7f6f5cf989a1/supabase/migrations/20230815171919_base.sql#L1872) and [this line](https://github.com/Cap-go/capgo/blob/2210c90ae8c9dd957914d658b37e7f6f5cf989a1/supabase/migrations/20230815171919_base.sql#L1887). Since you cannot have 2 constrains that are exactly the same I had to remove this line